modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -5425,6 +5427,9 @@ public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, S
                             LOGGER.warn("codegenModel is null. Default to UNKNOWN_BASE_TYPE");
                             codegenModelName = "UNKNOWN_BASE_TYPE";
                             codegenModelDescription = "UNKNOWN_DESCRIPTION";
+                            if (useOneOfInterfaces && templateDir.equals("JavaSpring")){
```

Comment:
I'm not familiar with JavaSpring generator, so could you please explain this piece of code a bit more? Can you provide an example of what it solves?

Reply:
I wanted here to be sure that, the changes in code will be executed only for spring.